app/api/questions/[id]/comments/route.ts

```
@@ -0,0 +1,212 @@
+    if (comments.length === 0) {
+      return NextResponse.json([])
+    }
+
+    // Fetch user metadata from Supabase Auth using direct SQL query
+    const userMetadataMap = new Map<string, { displayName: string | null; avatarUrl: string | null }>()
+    const uniqueUserIds = Array.from(new Set(comments.map(c => c.userId)))
+    
+    if (uniqueUserIds.length > 0) {
+      try {
+        // Query auth.users table directly via Prisma raw SQL
+        const authUsers = await prisma.$queryRaw<Array<{ id: string; raw_user_meta_data: any }>>`
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)